executor/executor_test.go

```diff
@@ -1872,6 +1875,34 @@ func (s *testSuite) TestIsPointGet(c *C) {
 	}
 }
 
+func (s *testSuite) TestPointGetRepeatableRead(c *C) {
+	c.Assert(failpoint.Enable("github.com/pingcap/tidb/executor/pointGetRepeatableReadTest", `return(true)`), IsNil)
+	tk1 := testkit.NewTestKit(c, s.store)
+	tk1.MustExec("use test")
+	tk1.MustExec(`create table point_get (a int, b int, c int,
+			primary key k_a(a),
+			unique key k_b(b))`)
+	tk1.MustExec("insert into point_get values (1, 1, 1)")
+	tk2 := testkit.NewTestKit(c, s.store)
+	tk2.MustExec("use test")
+
+	ctx := context.WithValue(context.Background(), "pointGetRepeatableReadTest", true)
```

Comment:
you can use failpoint.WithHook, then you can remove `ctx.Value("pointGetRepeatableReadTest") != nil` check:

```
	failpoint.Inject("pointGetRepeatableReadTest", func(val failpoint.Value) {
			if val.(bool) && ctx.Value("pointGetRepeatableReadTest") != nil {
				label := debugger.Bind("point-get-g1")
				debugger.Break(label)
			}
		})
```

Reply:
Done